Sandbox.Dockerfile

```
@@ -0,0 +1,13 @@
+
+RUN apk add --no-cache python3=3.12.10-r0 make=4.4.1-r2 g++=14.2.0-r4 curl=8.12.1-r1 \
+    && npm install -g pnpm@10.6.5
+
+WORKDIR /app
+
+COPY package.json pnpm-lock.yaml checkNodeVersion.js ./
+
+COPY scripts ./scripts
+RUN chmod +x ./scripts/build-out.sh
+
+RUN pnpm install
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)